about.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="a website about ferrari racing events" />
+    <meta name="author" content="Arthur Emanuel Guerra Iturres" />
+    <link rel="icon" href="src/assets/media/icons-logos/ferrari-logo-png.png" />
+    <link
+      rel="apple-touch-icon"
+      href="src/assets/media/icons-logos/ferrari-logo-png.png"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="src/assets/styles/about.css" />
+    <title>Ferrari | About</title>
+  </head>
+  <body>
+    <header class="my-header">
+      <nav class="desktop-navbar navbar navbar-dark justify-content-end p-2">
+        <ul class="d-flex list-unstyled container">
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done but in the options in the `navigation menu` there is an option to go to the `"About"` page but there is no option to go back to `"Home"` in the menu, please fix it 🙏🏼 

![image](https://github.com/ITurres/Ferrari-Racing-Website/assets/63915024/d64cb025-fea3-492a-b88a-1e66c6f3d96e)

Reply:
Hello again @mukhammadrizooff 😁!, Regarding the mobile navigation, I overlooked it 😄, thank you! Yes, I will change the ‘about’ to ‘home’ and point the link to the respective page right away.